### ISSUE TITLE
Make cd ..... easy

### ISSUE THREAD

**Issue by 0mp** [0mp/goat#17]
Let goat detect `cd ...` and resolve it to `cd ../..`. Make it work for any number of dots.

**Comment by GaryLeutheuser** [comment]
I'm starting work on this now!

**Comment by 0mp** [comment]
@GaryLeutheuser That's awesome! Thanks a lot!

**Comment by 0mp** [comment]
The functionality has been implemented in #24. 

Todo:
- [ ] Add info about it to README.
- [ ] Specify the shortcut resolution order. I'd go for:
  - Check if the shortcut is local, for example:
    
    ``` sh
    user@pc:/$ goat usr /etc
    user@pc:/$ cd usr
    # You don't want to follow the usr shortcut to /etc. 
    # You want to enter /usr instead.
    user@pc:/usr$
    ```
  - Check the standard shortcuts file.
  - Check the relative shotcuts file (not merged into master yet).
  - Check if the shortcut is all dots.
  - Throw an error
